<launch>
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyARDU"/>
    <param name="baud" value="57600"/>
    <param name="auto_pilot_mode" value="0"/>
    <param name="led_enable" value ="0"/>
    <param name="Right_motor" value = "1610"/>
    <param name="Left_motor" value = "1550"/>
  </node>
</launch>
